libs/utils/src/lib/serializer.service.spec.ts

```diff
@@ -27,4 +28,64 @@ describe('Serializer', () => {
       expect(serialized).toEqual(['--', '--one=two']);
     });
   });
+
+  describe('normalizeSchematic', () => {
+    const schematic: Schematic = {
+      collection: '@nrwl/testing',
+      name: 'normalized',
+      description: 'a schematic to be normalized.',
+      schema: [
+        {
+          type: 'arguments',
+          name: 'required',
+          enum: [],
+          description: '',
+          defaultValue: null,
+          required: true,
+          positional: false,
+          important: false
+        },
+        {
+          type: 'arguments',
+          name: 'positional',
+          enum: [],
+          description: '',
+          defaultValue: null,
+          required: false,
+          positional: true,
+          important: false
+        },
+        {
+          type: 'arguments',
+          name: 'ignoreImportant',
+          enum: [],
+          description: '',
+          defaultValue: null,
+          required: false,
+          positional: false,
+          important: true
+        }
+      ]
+    };
+    let normalized: Schematic;
+    it('should normalize the schematic, removing periods from the end of description', () => {
+      normalized = serializer.normalizeSchematic(schematic);
+      expect(normalized.description.lastIndexOf('.')).toBeLessThan(
+        normalized.description.length
+      );
+    });
+    it('should recognize positional and required fields as important but ignore previous values of important', () => {
+      normalized.schema.forEach((field: Field) => {
```

Comment:
You've written your tests in such a way that the order they are executed in matters. Could you instead use a beforeEach function which instantiates `normalized` rather than relying on the previous test?

Reply:
I'm going to lead with "I didn't do the specific thing you asked me to do." I accomplished the same goal, but I didn't want to use a beforeEach to instantiate the result of a method call. I didn't want to have it quietly increase test execution time as new test cases are added when we really only care about testing that one value.